config_builder.go

```diff
@@ -1475,27 +1478,27 @@ func parseHeaderStateAssertion(state string) (*headerfs.FilterHeader, error) {
 // the neutrino BroadcastError which allows the Rebroadcaster which currently
 // resides in the neutrino package to use all of its functionalities.
 func broadcastErrorMapper(err error) error {
-	returnErr := rpcclient.MapRPCErr(err)
+	var returnErr error
```

Comment:
Why not map the error here like before?

Reply:
To use the method `MapRPCErr` we need to access the chain backend, which cannot be used here. Instead, we pass the already mapped error to `broadcastErrorMapper`, and access the method via `cs.MapRPCErr` above.